httplivestreaming.c

```diff
@@ -347,9 +315,6 @@ int hls_write_packet(HTTPLiveStreaming *hls, AVPacket *pkt, int split) {
     for (i = 0; i < nb_streams; i++) {
       ts_st = hls->format_ctx->streams[i]->priv_data;
       stream_cc[i] = ts_st->cc;
-      if (i == 0) {
```

Comment:
Note I don't really know what these 3 lines or the 3 lines below (369-371) were for.  They just had to be removed because there's no references to a `service` structure in `ts_st`, so I just removed them.  Feel free to enlighten me is this is incorrect.

Reply:
That code was to retain the value of continuity counter for Program Map Table (PID 4096) between two consecutive *.ts files. I've added the code for it at commit 6684cfe11534f86a9b47556b48a2632b176e26ab.